frontend/src/lib/components/TraineeList.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+	import { writable } from 'svelte/store';
+	import { UserRoundPlus } from 'lucide-svelte';
+
+	let trainees = $props()
+
+	// dummy data for trainees
+	trainees = writable([
+		{ uuid: '1', username: 'john.doe@example.com', name: 'Garbor', surname: 'John' },
```

Comment:
not the right place for hardcoded data. Think of the flow how it happens with an API client and then replace the asynchronous call of the api client with the hardcoded data. this is not possible here